src/libraries/System.IO.FileSystem/src/System/IO/EnumerationOptions.cs

```diff
@@ -34,6 +34,7 @@ public EnumerationOptions()
         {
             IgnoreInaccessible = true;
             AttributesToSkip = FileAttributes.Hidden | FileAttributes.System;
+            MaxRecursionDepth = int.MaxValue;
```

Comment:
You should use the named constant here as well.

Reply:
Done.